### PR TITLE
clean up lastTaskFailure for schema validation

### PR DESCRIPTION
**Pull request by tamarrow**
This fixes bug found by intercom user: https://app.intercom.io/a/apps/wn4z9z0y/inbox/conversation/997828033

### CODE REVIEW

**Comment by tamarrow** [comment]
bump

File: cli/dcoscli/marathon/main.py [hunk 1]
```diff
@@ -656,7 +656,7 @@ def _clean_up_resource_definition(properties):
         if v:
             if k in ["apps", "groups"]:
                 clean_properties[k] = [_clean_up_resource_definition(v[0])]
-            elif not k.startswith("task"):
+            elif not k.startswith("task") and k != "lastTaskFailure":
```
Comment (jsancio) [inline cli/dcoscli/marathon/main.py]: Let's add a comment as to why we are doing this. A comment for both `...startswith("task")` and `... "lastTaskFaillure"` I think I have ask the same question more than once already.

**Comment by jsancio** [comment]
Looks good. :shipit:. One small comment.

File: cli/dcoscli/marathon/main.py (hunk 1 above)
Comment (tamarrow) [inline cli/dcoscli/marathon/main.py]: done

**Pull request merged**